cms/templates/js/xblock-string-field-editor.underscore

```diff
@@ -1,16 +1,16 @@
 <div class="incontext-editor-action-wrapper">
-    <a href="" class="action-edit action-inline xblock-field-value-edit incontext-editor-open-action" title="<%- gettext('Edit the name') %>">
-        <span class="icon fa fa-pencil" aria-hidden="true"></span><span class="sr"> <%- gettext("Edit") %></span>
-    </a>
+    <button type="button" class="action-edit action-inline xblock-field-value-edit incontext-editor-open-action" title="<%- gettext('Edit the name') %>">
+        <span class="icon fa fa-pencil" aria-hidden="true"></span>
+    </button>
 </div>
 
 <div class="xblock-string-field-editor incontext-editor-form">
   <form>
     <% var formLabel = gettext("Edit %(display_name)s (required)"); %>
-    <label><span class="sr"><%= interpolate(formLabel, {display_name: fieldDisplayName}, true) %></span>
-      <input type="text" value="<%= value %>" class="xblock-field-input incontext-editor-input" data-metadata-name="<%= fieldName %>" title="<%= gettext('Edit the name') %>">
+    <label><span class="sr"><%- HtmlUtils.HTML(interpolate(formLabel, {display_name: fieldDisplayName}, true)) %></span>
```

Comment:
There's a test that checks for various HTML characters (<, >, single, and double quotes) so this is necessary. Or I remove that test. Which would you prefer?